rust_snuba/src/processors/functions.rs

```diff
@@ -100,7 +100,6 @@ struct Function<'a> {
     http_method: Option<&'a str>,
     is_application: u8,
     materialization_version: u8,
-    module: &'a str,
```

Comment:
how come the rust processor is changing too?

Reply:
This field is unused and was picked up by the Rust processor when the Python processor wasn't ingesting it.